python/py-gpstime/Portfile

```diff
@@ -29,18 +27,22 @@ homepage            https://git.ligo.org/cds/${python.rootname}
 
 master_sites        https://git.ligo.org/cds/${python.rootname}/-/archive/${version}/
 distname            ${python.rootname}-${version}
-extract.suffix      .tar.gz
 
 checksums           rmd160  d62957cd15570ce24e5e17d5fe6d587777698809 \
                     sha256  b303807ba1e6a93f1ee1d47f36c014f2580348dd5045dd0cb28c1a59a0d700aa \
                     size    23900
 
-python.versions         35 36
+python.versions     35 36 37
 
-python.default_version  36
+livecheck.type  none
```

Comment:
please leave the ```livecheck.type none``` within the ```if {${name} ne ${subport}}``` block as is done for (almost) all Python ports; your original was correct.